clang/lib/Sema/SemaDecl.cpp

```diff
@@ -19213,6 +19213,29 @@ void Sema::ActOnFields(Scope *S, SourceLocation RecLoc, Decl *EnclosingDecl,
 
     if (Record && FD->getType().isVolatileQualified())
       Record->setHasVolatileMember(true);
+    auto IsNonDependentBitField = [](const FieldDecl *FD) {
+      if (!FD->isBitField())
+        return false;
+      if (FD->getType()->isDependentType())
+        return false;
+      return true;
```

Comment:
```suggestion
      return !FD->getType()->isDependentType();
```

Reply:
Honestly, I’d even do this:
```c++
auto IsNonDependentBitField = [](const FieldDecl *FD) {
  return FD->isBitField() && !FD->getType()->isDependentType();
};
```
(I would have loved to format that as suggestion, but github isn’t letting me because this is in a reply...)

Reply:
I think I did this intentionally as I sometimes feel like explicit true/false returns are clearer, I'm happy with either Aaron or @Sirraide's suggestion but I want to confirm that others don't have my clarity concerns :D